torch_xla/csrc/runtime/runtime.h

```diff
@@ -3,10 +3,19 @@
 
 #include "torch_xla/csrc/runtime/computation_client.h"
 
-namespace torch_xla {
-namespace runtime {
+namespace torch_xla::runtime {
+namespace status {
 
 // Returns the ComputationClient singleton.
+absl::StatusOr<ComputationClient*> GetComputationClient();
+
+}
+
+ABSL_DEPRECATED(
+    "Use status::GetComputationClient(), instead. "
+    "This function throws an exception on error, instead of "
+    "actually handling the StatusOr return value, which is "
+    "safer.")
 ComputationClient* GetComputationClient();
```

Comment:
Did you see my comments on the issue about renaming this to `*OrDie()` and having the new function in the `torch_xla::runtime` namespace?

Reply:
Yeah. I did update as you suggested. I think you reviewed an outdated version.